config/rbac/topolvm_node_scc.yaml

```diff
@@ -0,0 +1,30 @@
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: topolvm-node
+  annotations:
+    kubernetes.io/description: 'privileged allows access to all privileged and host
+        features and the ability to run as any user, any group, any fsGroup, and with
+        any SELinux context.  WARNING: this is the most relaxed SCC and should be used
+        only for cluster administration. Grant with caution.'
+allowHostDirVolumePlugin: true
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: true
+allowHostPorts: false
+allowPrivilegeEscalation: true
```

Comment:
I don't think we need privilegeEscalation

Reply:
@nbalacha : We cannot use  role instead of cluster role, because we need permissions for cluster-scoped resources (like nodes).

Reply:
it is needed because we use "privileged" containers:  if we change this setting , we have errors deploying the daemonset, example:

` Warning  FailedCreate  44s               daemonset-controller  Error creating: Pod "topolvm-node-qx67b" is invalid: [spec.containers[0].securityContext: Invalid value: core.SecurityContext{Capabilities:(*core.Capabilities)(nil), Privileged:(*bool)(0xc0571abd0a), SELinuxOptions:(*core.SELinuxOptions)(nil), WindowsOptions:(*core.WindowsSecurityContextOptions)(nil), RunAsUser:(*int64)(0xc0571abd60), RunAsGroup:(*int64)(nil), RunAsNonRoot:(*bool)(nil), ReadOnlyRootFilesystem:(*bool)(nil), AllowPrivilegeEscalation:(*bool)(0xc0571abc48), ProcMount:(*core.ProcMountType)(nil), SeccompProfile:(*core.SeccompProfile)(nil)}: cannot set `allowPrivilegeEscalation` to false and `privileged` to true`

Reply:
ack.